src/components/Movie.jsx

```diff
@@ -0,0 +1,72 @@
+import { useDispatch, useSelector } from 'react-redux'
+import starredSlice from '../data/starredSlice'
+import watchLaterSlice from '../data/watchLaterSlice'
+import placeholder from '../assets/not-found-500X750.jpeg'
+
+const Movie = ({ movie, viewTrailer, closeCard }) => {
+
+    const state = useSelector((state) => state)
+    const { starred, watchLater } = state
+    const { starMovie, unstarMovie } = starredSlice.actions
+    const { addToWatchLater, removeFromWatchLater } = watchLaterSlice.actions
```

Comment:
I think we can import all actions from the slices directly instead of the whole slice like this:
`export const {  addToWatchLater,  removeFromWatchLater } = watchLaterSlice.actions`
 and bring it up to the top of the file